src/Application/Controller/Projects.php

```diff
@@ -114,6 +118,25 @@ public function profiles() {
 						);
 					}
 				}
+				
+				if (!empty($values['auto_create']) and
+					is_array($values['auto_create'])) {
+					foreach ($values['auto_create'] as $id => $auto_create) {
```

Comment:
We normally use camel case for variables (except in XMLRPC_Handler).

Reply:
Okay.